src/app/features/bird-info/columns.ts

```diff
@@ -0,0 +1,24 @@
+import { DataGridCol } from "../../../components/table/DataGrid";
+import { COMMON_GRID_COLUMNS } from "../../../utils/grid/commonGridColumns";
+import { GridColumn } from "../../../utils/grid/GridColumn";
+
+export const BIRD_OBSERVATION_LIST_COLUMNS_CONFIG: DataGridCol<any>[] = [
```

Comment:
maybe would be a good idea to import and reuse OBSERVATION_LIST_COLUMNS_CONFIG here, since i don't think there would be big differences in columns

Reply:
fixed